src/esp/bindings_js/index.js

```diff
@@ -1,12 +1,26 @@
-/*global Module */
+/* global FS, Module */
 
 import WebDemo from "./modules/web_demo";
 import VRDemo from "./modules/vr_demo";
 import "./bindings.css";
 
-Module["onRuntimeInitialized"] = function() {
-  console.log("hsim_bindings initialized");
+Module.preRun.push(() => {
+  let arg;
+  let scene;
```

Comment:
NIT: To make it clear that this is url, maybe name it sceneUrl?

Reply:
Its also the filename we are using in the virtual filesystem.